ioSPI/transfer.py

```diff
@@ -0,0 +1,174 @@
+import numba as nb
+import numpy as np
+
+
+def ctf_freqs(N, psize=1.0, d=2):
```

Comment:
lower case, bcs N is not a constant, see:  https://www.educative.io/edpresso/what-are-constants-in-python

also, `n_pixels` would be better (again, avoiding single letter for variable names will help save time if we want to automatically find and replace this variable one day)

Reply:
good point about one letter variables